.vsts-ci.yml

```diff
@@ -100,6 +100,12 @@ stages:
         steps:
         - checkout: self
           clean: true
+        # Scan markdown files on style consistency
+        - powershell: |
```

Comment:
Looks like this step takes about between 30 s and a minute, which for now is fine. If we expect it to take longer, then we should consider disabling the step for official builds and only running on CI.

Reply:
That's correct, it can only run for PR. As once in the repo, it should be clean. And there is no need to run it for release.
So any recommendation on the best way to adjust this?